unity/PitayaExample/Assets/Pitaya/EventManager.cs

```diff
@@ -115,7 +115,8 @@ public void Dispose()
         // ReSharper disable once UnusedParameter.Local
         private void Dispose(bool disposing)
         {
-			foreach (var callback in _errorCallBackMap)
+            var errorCallBackMap = new Dictionary<uint, Action<PitayaError>>(_errorCallBackMap);
+			foreach (var callback in errorCallBackMap)
 				callback.Value.Invoke(new PitayaError(PitayaConstants.PitayaInternalError, "pitaya exited"));
```

Comment:
Could you fix the indenting here?

Reply:
Done! For some reason, only the loop block was using tabs in this file xD.